osi_sensorviewconfiguration.proto

```diff
@@ -752,27 +752,126 @@ message CameraSensorViewConfiguration
+        // Bayer BGGR Channels UINT16 FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_U16_LIN = 11;
 
-        // Bayer RGGB Channels UINT32 FP Linear.
+        // Bayer BGGR Channels UINT32 FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_U32_LIN = 12;
 
-        // Bayer RGGB Channels Single Precision FP Linear.
+        // Bayer BGGR Channels Single Precision FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_F32_LIN = 13;
 
+        // Bayer RGGB Channels UINT8 FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_U8_LIN = 14;
+
+        // Bayer RGGB Channels UINT16 FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_U16_LIN = 15;
+
+        // Bayer RGGB Channels UINT32 FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_U32_LIN = 16;
+
+        // Bayer RGGB Channels Single Precision FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_F32_LIN = 17;
+
+        // Read Clear Clear Clear Channels UINT8 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCC_U8_LIN = 18;
+
+        // Read Clear Clear Clear Channels UINT16 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCC_U16_LIN = 19;
+
+        // Read Clear Clear Clear Channels UINT32 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCC_U32_LIN = 20;
+
+        // Read Clear Clear Clear Channels Single Precision FP Linear.
+        //
+        CHANNEL_FORMAT_RCCC_F32_LIN = 21;
+
+        // Read Clear Clear Blue Channels UINT8 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCB_U8_LIN = 22;
+
+        // Read Clear Clear Blue Channels UINT16 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCB_U16_LIN = 23;
+
+        // Read Clear Clear Blue Channels UINT32 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCB_U32_LIN = 24;
+
+        // Read Clear Clear Blue Channels Single Precision FP Linear.
+        //
+        CHANNEL_FORMAT_RCCB_F32_LIN = 25;
+        
+
         // TBD: Further channel permutations and padding (e.g. RGBZ,
         // BGR, BAYER_RGGB/GBRG/GRBG/...), non-BAYER filters, non-linear
         // encodings, ...
     }
 
+    // Number of samples per pixel.
+    //
+    // \note This is a characteristic of the ray tracing engine of the
+    // environment simulation, not a direct characteristic of the sensor.
+    //
+    // \rules
+    // is_greater_than_or_equal_to: 1
+    // \endrules
+    //
+    optional uint32 samples_per_pixel = 9;
+
+    // Maximum number of interactions to take into account.
+    //
+    // \note This is a characteristic of the ray tracing engine of the
+    // environment simulation, not a direct characteristic of the sensor.
+    //
+    // \rules
+    // is_greater_than_or_equal_to: 1
+    // \endrules
+    //
+    optional uint32 max_number_of_interactions = 10;
+
+    // In use-cases where a spectral ray-tracer is used, this message determines the range of the wavelength
+    // and its desired number of samples.
+    //
+    optional WavelengthData wavelength_data = 11;
```

Comment:
Might this not also be a repeated field, to allow multiple specifications, for e.g. each filter wavelength range...

Reply:
Updated to repeated